src/fairseq2/recipes/lm/text_generate.py

```diff
@@ -332,26 +349,32 @@ class TextGenerateUnit(AbstractGeneratorUnit[SequenceBatch]):
 
     _generator: SequenceGenerator
     _text_decoder: TextTokenDecoder
-    _output_stream: Optional[TextIO]
+    _output_txt_txt_stream: Optional[TextIO]
+    _output_jsonl_txt_stream: Optional[TextIO]
     _metric_bag: SequenceGenerationMetricBag
 
     def __init__(
         self,
         generator: SequenceGenerator,
         tokenizer: TextTokenizer,
         gang: Gang,
-        output_stream: Optional[TextIO],
+        output_txt_txt_stream: Optional[TextIO],
```

Comment:
I suggest naming these: `output_text_stream` and `output_json_stream`. In particular, `output_txt_txt_stream` sounds a little off to me.

Reply:
oh that sounds like a bad ctrf-f -> replace :)